src/internet_identity/Cargo.toml

```diff
@@ -5,9 +5,8 @@ edition = "2018"
 
 [dependencies]
 base64 = "0.13.0"
-certified_map = { path = "../certified_map" }
+ic-certified-map = "0.2.0"
```

Comment:
This version is the newest on crates.io, but it would be great to move to the one where the unsafe code is removed. Are you planning to do this before merging?

Reply:
nit: we usually sort the dependencies in alphabetical order. Could you do that too?

Reply:
out of curiosity, which one is the one without the unsafe code? is that on GitHub but not yet released?

re. alphabetical order: would be worth checking this and doing it automatically! wdyt?

Reply:
Sorted imports. Referenced latest version via commit hash.

> re. alphabetical order: would be worth checking this and doing it automatically! wdyt?

This would be great.